src/store/registration/saga.ts

```diff
@@ -169,6 +172,30 @@ export function* updateProfile(action) {
       yield put(setFirstTimeLogin(true));
       yield call(completeUserLogin);
       yield put(setStage(RegistrationStage.Done));
+
+      const inviterMatrixId = response.response.inviter.matrixId;
+      const inviterUserId = response.response.inviter.id;
+      const userData = yield call(getZEROUsersAPI, [inviterMatrixId]);
+
+      const normalizeUserData = (userData) => {
+        return {
+          userId: userData.id,
+          firstName: userData.profileSummary.firstName,
+          profileImage: userData.profileSummary.profileImage,
+          matrixId: userData.matrixId,
+        };
+      };
+
+      if (userData && userData.length > 0) {
+        yield put(receive({ users: { [inviterUserId]: normalizeUserData(userData[0]) } }));
```

Comment:
This feels a little off. We shouldn't generally be forcefully "receiving" directly from the normalized data. Pretty sure you want to import `receive` from the `users` store and just receive the user:
`yield put(receive(<user>));`

At least, that's my hunch.

Reply:
Updated this, thanks!